From 18458efc977a1c52bf2e896b56f20f51f4fafc1c Mon Sep 17 00:00:00 2001
From: Haibo Chen <haibo.chen@freescale.com>
Date: Wed, 20 May 2015 14:28:08 +0800
Subject: [PATCH 1111/1594] MLK-10912-5 ARM: dts: imx6ul-evk.dts: add tsc
 support

commit d302a870a5d852744e2a7435a305d6edffae2373 from
git://git.freescale.com/imx/linux-2.6-imx.git

Add touch screen support for i.MX6UL-EVK board.

Signed-off-by: Haibo Chen <haibo.chen@freescale.com>
---
 arch/arm/boot/dts/imx6ul-evk.dts |   18 ++++++++++++++++++
 1 files changed, 18 insertions(+), 0 deletions(-)

diff --git a/arch/arm/boot/dts/imx6ul-evk.dts b/arch/arm/boot/dts/imx6ul-evk.dts
index a6795b9..d46e62f 100755
--- a/arch/arm/boot/dts/imx6ul-evk.dts
+++ b/arch/arm/boot/dts/imx6ul-evk.dts
@@ -243,6 +243,15 @@
 	status = "okay";
 };
 
+&tsc {
+	pinctrl-names = "default";
+	pinctrl-0 = <&pinctrl_tsc>;
+	status = "okay";
+	xnur-gpio = <&gpio1 3 0>;
+	measure_delay_time = <0xffff>;
+	pre_charge_time = <0xfff>;
+};
+
 &uart1 {
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_uart1>;
@@ -502,6 +511,15 @@
 			>;
 		};
 
+		pinctrl_tsc: tscgrp {
+			fsl,pin = <
+				MX6UL_PAD_GPIO1_IO01__GPIO1_IO01	0xb0
+				MX6UL_PAD_GPIO1_IO02__GPIO1_IO02	0xb0
+				MX6UL_PAD_GPIO1_IO03__GPIO1_IO03	0xb0
+				MX6UL_PAD_GPIO1_IO04__GPIO1_IO04	0xb0
+			>;
+		};
+
 		pinctrl_i2c1: i2c1grp {
 			fsl,pins = <
 				MX6UL_PAD_UART4_TX_DATA__I2C1_SCL 0x4001b8b0
-- 
1.7.5.4

